Hana/src/Hana/OrthographicCameraController.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "Hana/Renderer/OrthographicCamera.h"
+#include "Hana/Core/Timestep.h"
+
+#include "Hana/Events/ApplicationEvent.h"
+#include "Hana/Events/MouseEvent.h"
+
+namespace Hana
+{
+	class OrthographicCameraController
+	{
+	public:
+		OrthographicCameraController(float aspectRatio, bool rotation = false);
+
+		void OnUpdate(Timestep ts);
+		void OnEvent(Event& e);
+
+		OrthographicCamera& GetCamera() { return m_Camera; }
+		const OrthographicCamera& GetCamera() const { return m_Camera; }
+	private:
+		bool OnMouseScrolled(MouseScrolledEvent& e);
+		bool OnWindowResized(WindowResizeEvent& e);
+
+		float m_AspectRatio = 1.7f;
+		float m_ZoomLevel = 1.0f;
+		OrthographicCamera m_Camera;
```

Comment:
The order of member variables does matter here. We need m_AspectRation and m_ZoomLevel to be initialized first to create the m_Camera.